data/strings/sound.txt

```diff
@@ -504,7 +666,7 @@
     tr = Hedef noktasına vardınız.
     uk = Ви прибули.
     vi = Bạn đã đến.
-    zh-Hans = 您已到达目的地。
+    zh-Hans = 您已到达目的地
```

Comment:
Dots were removed intentionally, right? If yes (and if it sounds better without a dot in Chinese), then please also remove dots in corresponding zh-Hant translations.

Reply:
I'll just re-add them. If it makes a difference for some people's TTS then we can keep them, my code handles them properly. This was just not-caring more than intention.